internal/cli/filtering_test.go

```diff
@@ -168,7 +168,7 @@ func TestExpandNamedFiltersJoinMatch(t *testing.T) {
 	)
 
 	assert.Nil(t, err)
-	assert.Equal(t, res.Match, "(((chocolate OR caramel)) AND (banana)) AND (apple)")
+	assert.Equal(t, res.Match, []string{"(chocolate OR caramel)", "banana", "apple"})
```

Comment:
Implicit AND now, so had to alter test case expectation

Reply:
Looks good.